source/math/FloatComplexVectorList.ooc

```diff
@@ -161,4 +174,32 @@ FloatComplexVectorList: class extends VectorList<FloatComplex> {
 			result add(value)
 		result
 	}
+	_fastFourierTransformBufferSize: static func (inputSize: Int) -> Int {
+		logValue := inputSize as Float log() / 2.0f log()
```

Comment:
I guess it could be useful. Maybe a separate pull request with missing *Math* functions (log2, log10) ? Can you think about any more functions which are missing and could be useful ?